modules/cluster/aws_ebs_csi_driver_iam.tf

```
@@ -23,7 +23,7 @@ data "aws_iam_policy_document" "aws_ebs_csi_driver_assume_role_policy" {
 
 resource "aws_iam_role" "aws_ebs_csi_driver" {
   count                = local.aws_ebs_csi_driver_iam_role_count
-  name                 = "EksEBSCSIDriver-${var.name}"
+  name                 = "${var.iam_role_name_prefix}EksEBSCSIDriver-${var.name}"
```

```suggestion
  name                 = "${var.iam_role_name_prefix}-EksEBSCSIDriver-${var.name}"
```
would be nice for readability - or do you need this as-written to be compatible with with what you've implemented for the web platform clusters?

---

I think this makes the prefix non optional (default role name will be `-EksEBSCSIDriver-nanana`)?

---

Perhaps we can do something like
```suggestion
  name                 = join("-", compact([var.iam_role_name_prefix, "EksEBSCSIDriver", var.name]))
```

---

ah yes neater than the suggestion I was mid-writing with a var null condition